web/src/pages/sbom.tsx

```diff
@@ -0,0 +1,12 @@
+import ExternalLink from "@site/src/components/ExternalLink";
+import React from "react";
+
+const Element = () => {
+  return (
+    <ExternalLink
+      url={"https://github.com/tenzir/tenzir/releases/latest/download/Tenzir.spdx"}
```

Comment:
This link is 404 for me.

Reply:
Right. The reason is that VAST v3.1 was the `latest` release. RCs do not count as `latest`, so this link will only work as of Tenzir v4.0.

Reply:
If you take a look at the current RCs, `Tenzir.spdx` already exists there.